examples/by_feature/fsdp_with_peak_mem_tracking.py

```diff
@@ -162,6 +162,7 @@ def collate_fn(examples):
     # New Code #
     # For FSDP feature, it is highly recommended and efficient to prepare the model before creating optimizer
     model = accelerator.prepare(model)
+    accelerator.print(model)
```

Comment:
Is this supposed to stay here?

Reply:
Yes, to help user observe which layers have been auto-wrapped in FSDP modules.